src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1347,7 +1346,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                     !(_terminal->GetHyperlinkAtPosition(terminalPosition).empty()))
                 {
                     _HyperlinkHandler(_terminal->GetHyperlinkAtPosition(terminalPosition));
+                    args.Handled(true);
+                }
+
+                if (args.Handled())
+                {
+                    // We already handled the event by either sending it to the VT,

```

Comment:
```suggestion
                    // GH#9117: We already handled the event by either sending it to the VT,
```
nit: link the issue for easier tracking